spec/fixtures/generate_crt_and_Key_for_1028_keylength.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
It seems out of place for the K in Key for the filename to be the only capital character. Intentional?